tests/test_cameraFilter.py

```diff
@@ -1,8 +1,15 @@
-
-from vgc.cameraFilter import cameraFilter 
+from time import sleep
+from vgc.camera_filter import CameraFilter 
 
 def test_camerafilter():
-    cf = cameraFilter([])
+    """Testing the Came"""
```

Comment:
Try to explain more clearly what this fucntion actually tests.

Reply:
The tests for the camera filter are not done and therefore not pushed.